examples/MyModule/soundclick_example.py

```diff
@@ -0,0 +1,41 @@
+import nest
```

Comment:
Please create a new directory "examples" in the MyModule directory and move this file there. I consider it to be not very clean to have an example in the toplevel directory.

Reply:
Also, this file should have a copyright header and documentation as the other PyNEST examples do.

Reply:
This file also needs a copyright header and documentation in the style of the other PyNEST examples.

Reply:
Done!